### ISSUE TITLE
Feature suggestion: boolean scopes

### ISSUE THREAD

**Issue by sophiedeziel** [BaseSecrete/type_scopes#1]
I'm leaving here an example of how I picture it. Should be self-explanatory

```Ruby
# paid_at: datetime
# amount: decimal
# description: string
# final: boolean
class Transaction < ActiveRecord::Base
  include TypeScopes
end

# Boolean scopes
Transaction.final # => where(final: true)
Transaction.final(false) # => where(final: false)
```

**Comment by alexisbernard** [comment]
Good idea. Maybe even `Transaction.not_final` :)

**Comment by alexisbernard** [comment]
It has been implemented in #2 .
